### ISSUE TITLE
[Bug] SQL insert failed with string value contains char `;`

### ISSUE THREAD

**Issue by zipper-meng** [cnosdb/cnosdb#2019]
### Describe the bug

Version : `cnosdb 2.3.5, revision 3fa5ee033ad3515f2bd01fb11d87c7fee7601ebe`

Execution of insert SQL statement by `cnosdb-cli` failed if there were string values that contain semicolon character: `;` in the SQL.

### To Reproduce

```sql
public ❯ create table foo(str_col string, tags(ta));
public ❯ describe table foo;
+-------------+-----------------------+-------------+-------------------+
| column_name | data_type             | column_type | compression_codec |
+-------------+-----------------------+-------------+-------------------+
| time        | TIMESTAMP(NANOSECOND) | TIME        | DEFAULT           |
| ta          | STRING                | TAG         | DEFAULT           |
| str_col     | STRING                | FIELD       | DEFAULT           |
+-------------+-----------------------+-------------+-------------------+
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', 'str');
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', 'str;');
422 Unprocessable Entity, details: {"error_code":"010009","error_message":"sql parser error: Unterminated string literal at Line: 1, Column 56"}
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', 'str;str');
422 Unprocessable Entity, details: {"error_code":"010009","error_message":"sql parser error: Unterminated string literal at Line: 1, Column 56"}
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', ';');
422 Unprocessable Entity, details: {"error_code":"010009","error_message":"sql parser error: Unterminated string literal at Line: 1, Column 56"}
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', ';str');
422 Unprocessable Entity, details: {"error_code":"010009","error_message":"sql parser error: Unterminated string literal at Line: 1, Column 56"}
public ❯ insert into foo(time, ta, str_col) values(now(), 'a1', ':str');
public ❯ select * from foo;
+----------------------------+----+---------+
| time                       | ta | str_col |
+----------------------------+----+---------+
| 2024-03-15T03:29:39.071008 | a1 | str     |
| 2024-03-15T03:30:09.514533 | a1 | :str    |
+----------------------------+----+---------+
```

### Expected behavior

All these executions were ok.

**Comment by Benxiaohai001** [comment]
Need to synchronize version 2.3

**Comment by Benxiaohai001** [comment]
The problem has been fixed, 2.3 has been synchronized, and use case has been added
